addons/api/tests/unit/models/worker-test.js

```diff
@@ -1,13 +1,45 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 
 module('Unit | Model | worker', function (hooks) {
   setupTest(hooks);
+  setupMirage(hooks);
 
   // Replace this with your real tests.
   test('it exists', function (assert) {
     let store = this.owner.lookup('service:store');
     let model = store.createRecord('worker', {});
     assert.ok(model);
   });
+
+  test('it has an `addWorkerLed` method that targets a specific POST API', async function (assert) {
+    assert.expect(1);
+    this.server.post(
+      '/v1/workers/123abc:create:worker-led',
```

Comment:
This `post` route doesn't look quite right, it's doing a `POST` to the individual worker `123abc` instead of the `workers` collection route. It should be a `POST` to `v1/workers`.

Reply:
fixed in next commit